packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1026,6 +1027,36 @@ class WidgetInspectorService {
       name: 'isWidgetCreationTracked',
       callback: isWidgetCreationTracked,
     );
+    assert(() {
```

Comment:
This method was previously reviewed just without the outer assert block to avoid the code from being included in release builds.